app/Http/Requests/Activity/Indicator/IndicatorRequest.php

```diff
@@ -27,6 +28,7 @@ class IndicatorRequest extends ActivityBaseRequest
     public function rules(): array
     {
         $data = request()->except(['_token']);
+        $data['url'] = $this->getRequestUri();
```

Comment:
check this code for xls import as they do not have desired url

Reply:
Removed line 31. residue code from previous implementation. extracting result id from url if not file upload can be found in line 66.